index.js

```diff
@@ -1,88 +1,125 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+var counter = 1
+console.log("%cQuestion 1", "color: red") 
+console.log(counter) 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+
+// => 2
+var counter = 2
+console.log(counter)
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
+
 
-// console.log(name)
+const name = 'Henry Velasquez'
```

Comment:
Great! the key here was using `const` because it doesn't allow us to reassign our variable